src/api/apiCore.cpp

```diff
@@ -398,6 +416,15 @@ void TapeCore::tape_texture_destroy(const YAML::Node& yamlNode, PlaybackState& s
 	state.textures.erase(textureId);
 }
 
+rgl_status_t rgl_is_texture_valid(rgl_texture_t texture, bool* out_valid)
+{
+	auto status = rglSafeCall([&]() {
+		*out_valid = Texture::instances.find(texture) != Texture::instances.end();
+	});
+	TAPE_HOOK();
+	return status;
+}
```

Comment:
```suggestion
rgl_status_t rgl_is_texture_valid(rgl_texture_t texture, bool* out_valid)
{
	auto status = rglSafeCall([&]() {
		CHECK_ARG(out_valid != nullptr);
		*out_valid = Texture::instances.contains(texture);
	});
	TAPE_HOOK(texture, out_valid);
	return status;
}
```

Reply:
Thank you. This has been fixed, see comment above.